utils/sccli.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright © 2016  Praveen Kumar <kumarpraveen.nitdgp@gmail.com>
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions
+# of the GNU General Public License v.2, or (at your option) any later
+# version.  This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY expressed or implied, including the
+# implied warranties of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.  You
+# should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# sccli - Service Change CLI for kubernetes, openshift and docker
+
+"""
+Standard Exit Codes (bash): http://tldp.org/LDP/abs/html/exitcodes.html
+Return Code:
+    110 - Service is not running
+    111 - Docker image not pulled
+
+Sample Usage:
+    $ python sccli.py -h
+    $ sudo python sccli.py kubernets stop
+    $ echo $? (if success return 0)
+    $ sudo python sccli.py openshift start
+    docker.io/openshift/origin:v1.1.9 Not Pulled (Message in stderr)
+    $ echo $?
+    111 (Exit code for Docker images pull issue)
+"""
+
+import os
+import subprocess
+import sys
+import time
+from argparse import ArgumentParser
+
+OPERATION = ['start', 'status', 'restart', 'stop']
+DOCKER_REGISTRY = "docker.io"
+IMAGE_NAME = "openshift/origin"
+IMAGE_TAG = "v1.1.3"
+
+
+def system(cmd):
+    """
+    Runs a shell command, and returns the output, err, returncode
+    :param cmd: The command to run.
+    :return:  Tuple with (output, err, returncode).
+    """
+    ret = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
+    out, err = ret.communicate()
+    returncode = ret.returncode
+    return out, err, returncode
+
+def service_status(service_name):
+    if service_name == "kubernetes":
+        status = 0
+        services = ['etcd', 'kube-apiserver', 'kube-controller-manager',
+                    'kube-scheduler', 'kube-proxy', 'kubelet']
```

Comment:
@LalatenduMohanty so list is not identical for both function if you look closely then for `status` we don't include docker service but in case of restart we use it.